mdoc/src/main/scala/mdoc/internal/markdown/MarkdownFile.scala

```diff
@@ -20,12 +21,26 @@ final case class MarkdownFile(input: Input, file: InputFile, parts: List[Markdow
   }
 }
 object MarkdownFile {
+  object syntax {
```

Comment:
moved syntax extensions to MarkdownFile instead of having all these in package object